### PR TITLE
[15.0][MIG] mgmtsystem_nonconformity

### PR DESCRIPTION
**Pull request by yankinmax**
Depends on:
 - [ ] https://github.com/OCA/management-system/pull/406

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+odoo-addon-document_page_procedure @ git+https://github.com/OCA/management-system@refs/pull/406/head#subdirectory=setup/document_page_procedure
```
Comment (i-vyshnevska) [inline test-requirements.txt]:
```suggestion
odoo15-addon-document_page_procedure @ git+https://github.com/OCA/management-system@refs/pull/406/head#subdirectory=setup/document_page_procedure
```

Comment (yankinmax) [reply test-requirements.txt]: we don't need 15 for odoo v15)))

**Review by ivantodorovich — APPROVED** [review]

**Comment by ivantodorovich** [comment]
Depended PR is merged, could you remove it from test-requirement ?

**Pull request merged**